exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/auth/DrillSpnegoAuthenticator.java

```diff
@@ -152,7 +153,7 @@ private Authentication authenticateSession(ServletRequest request, ServletRespon
         }
 
         response.setContentLength(0);
-        final HttpChannel channel = HttpChannel.getCurrentHttpChannel();
+        final HttpChannel channel = HttpConnection.getCurrentConnection().getHttpChannel();
         final Response base_response = channel.getResponse();
         final Request base_request = channel.getRequest();
```

Comment:
Please update this as below:
```
final Request base_request = Request.getBaseRequest(req);
final Response base_response = base_request.getResponse();
```

Reply:
Thanks, it looks better.
Also I have dropped usage of `base_response` local variable.